### PR TITLE
Updated composer to use latest version of Slim and Nexmo PHP SDK

### PR DESCRIPTION
**Pull request by GregHolmes**
I have upgraded the Slim and PHP SDK libraries. Also updated index.php to reflect these changes. 
The blog post doesn't require anything to be updated for this because the code example snippet hasn't been changed.

### CODE REVIEW

**Review by akrabat — COMMENTED** [review]
Just a couple of drive-by comments. Always nice to see Slim used for things like this 👍

File: composer.json [hunk 1]
```diff
@@ -1,7 +1,9 @@
 {
     "require": {
-        "slim/slim": "^3.12",
-        "slim/php-view": "^2.2",
-        "nexmo/client": "^1.6"
+        "nexmo/client": "^2.0",
+        "slim/slim": "^4.5",
+        "slim/psr7": "^1.1",
+        "slim/php-view": "^3.0",
+        "php-di/slim-bridge": "^3.0"
```
[the remark this answers is not in the document]
Comment (GregHolmes) [reply composer.json]: Thank you! I've removed the use of the container as it isn't really required for this project.

File: public/index.php [hunk 2]
```diff
@@ -1,61 +1,65 @@
...
+                $insight = $client->insights()->advanced($params['number']);
+                break;
+            default:
+                $insight = $client->insights()->basic($params['number']);
+                break;
+        }
 
+        if ($insight && $insight['status'] == 0) {
+            return $this->get('view')->render($response, 'main.php', ['insight' => $insight]);
+        }
```
[the remark this answers is not in the document]
Comment (GregHolmes) [reply public/index.php]: Thank you! The condition isn't required any more. If there's an error returned, it'll be an exception and caught.

**Review by lornajane — APPROVED** [review]
Such a solid update! Thanks @GregHolmes ! (and guest thanks to @akrabat for chiming in on this one!)

**Pull request merged**